### PR TITLE
628 detailed asset group

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- AssetGroup Create response contains detailed asset group sighting schema rather than basic
- Integration tests updated to only validate this once

---

**REST API Updates**

AssetGrpup Post response

```
{
    "asset_group_sightings": [
        {
            "asset_group_guid": "feace360-1afd-4fa2-bbc0-3f1873cb8510",
            "assets": [
                {
                    "annotations": [],
                    "created": "2021-12-21T09:42:33.423086+00:00",
                    "dimensions": {
                        "height": 664,
                        "width": 1000
                    },
                    "filename": "zebra.jpg",
                    "guid": "af22ddda-92a0-4792-b144-53ec7dfd83f0",
                    "src": "/api/v1/assets/src/af22ddda-92a0-4792-b144-53ec7dfd83f0",
                    "updated": "2021-12-21T09:42:33.423104+00:00"
                }
            ],
            "completion": 10,
            "config": {
                "assetReferences": [
                    "zebra.jpg"
                ],
                "encounters": [
                    {
                        "guid": "d0e07b32-ba8b-453b-9cd4-aea7aabb38a5"
                    }
                ],
                "locationId": "Lacock",
                "startTime": "never"
            },
            "creator": {
                "full_name": "First Middle Last",
                "guid": "1bdf3980-5363-4df2-927a-9b6e572bccab",
                "profile_fileupload": null
            },
            "curation_start_time": "2021-12-21T09:42:33.478038Z",
            "detection_start_time": null,
            "guid": "c3987b4b-d496-4bd5-8a95-2741f1e6e5fb",
            "jobs": {},
            "sighting_guid": null,
            "stage": "curation"
        }
    ],
    "assets": [
        {
            "filename": "zebra.jpg",
            "guid": "af22ddda-92a0-4792-b144-53ec7dfd83f0",
            "src": "/api/v1/assets/src/af22ddda-92a0-4792-b144-53ec7dfd83f0"
        }
    ],
    "commit": "a66cfadbed4ccde002598021c9e2c965203668df",
    "created": "2021-12-21T09:42:33.405317+00:00",
    "description": "This is a test asset_group, please ignore",
    "guid": "feace360-1afd-4fa2-bbc0-3f1873cb8510",
    "major_type": "filesystem",
    "owner_guid": "1bdf3980-5363-4df2-927a-9b6e572bccab",
    "updated": "2021-12-21T09:42:33.405326+00:00"
}
```

### CODE REVIEW

File: integration_tests/test_asset_group_sightings.py [hunk 1]
```diff
@@ -23,21 +23,21 @@ def test_asset_group_sightings(session, login, codex_url, test_root):
         {'commonNames': ['Example'], 'scientificName': 'Exempli gratia'},
     )
     tx_id = response.json()['response']['value'][-1]['id']
-    occ_test_cfd = utils.create_custom_field(
+    sighting_test_cfd = utils.create_custom_field(
```
Comment (hwindsor) [inline integration_tests/test_asset_group_sightings.py:26]: As agreed in discussion with @naknomum on last PR. The references to occurance are still required but minimise hem as much as possible

**Review by naknomum — APPROVED** [review]
LGTM

**Comment by naknomum** [comment]
new code to check assets exist look good.

**Review by naknomum — APPROVED** [review]
LGTM2

**Pull request merged**